mvc/models/model-binding.rst

```diff
@@ -1,8 +1,90 @@
-.. include:: /../common/stub-topic.txt
+Model Binding
+=============
 
-|stub-icon| Model Binding Request Data
-======================================
+By `Rachel Appel <http://github.com/rachelappel>`_
 
-.. include:: /../common/stub-notice.txt
+In this article
 
-.. _issue: https://github.com/aspnet/Docs/issues/114
+- `Introduction to model binding`_
+- `How model binding works`_
+- `Customize model binding behavior with attributes`_
+- `Binding formatted data from the request body`_ 
+
+Introduction to model binding
+-----------------------------
+Model binding in MVC maps data from HTTP requests to action method parameters. The parameters may be simple types such as strings, integers, or floats, or they may be class types. This is a great feature of MVC because mapping incoming data to a counterpart is an often repeated scenario, regardless of size or complexity of the data. MVC solves this problem by abstracting binding away so developers don't have to keep rewriting a slightly different version of that same code in every app. Writing your own text to type converter code is tedious, and error prone. 
+
+How model binding works
+-----------------------
+When MVC receives an HTTP request, it routes it to a specific action method of a controller. It determines which action method to run based on what is in the route data, then it binds values from the HTTP request to that action method's parameters. For example, consider the following URL:
+
+`http://contoso.com/movies/edit/2`
+
+Since the route template looks like this, ``{controller=Home}/{action=Index}/{id?}``, ``movies/edit/2`` routes to the ``Movies`` controller, and its ``Edit`` action method. It also accepts an optional parameter called ``id``. The code for the action method should look something like this: 
```

Comment:
The code for the action method -> The signature for the action method

I would then replace the literal include below with a code block that has just the action method signature (leave out the method body). You can then remove the entire sample project since it really isn't referenced at all.